nodejs/week1/meal_sharing_website/src/backend/routes/meals.js

```diff
@@ -1,9 +1,6 @@
 //Respond with the json for all the meals.
 //For each meal, if there are reviews matching it's meal ID, add these reviews to each meal in the form of an array. 
 //For meals that do not have any reviews, the "reviews" property will be an empty array.
-
-const express = require("express");
-const router = express();
 const meals = require("../data/meals.json");
 const reviews = require("../data/reviews.json");
 const mealReview = meals.map(meal => {
```

Comment:
rename to mealWithReview (mealReview is easily confused with a meal review i.e. a review of a meal)